gemfire-core/src/main/java/com/gemstone/gemfire/internal/snappy/CallbackFactoryProvider.java

```diff
@@ -50,6 +50,13 @@ public int getHashCodeSnappy(Object[] dvds) {
       throw new UnsupportedOperationException("unexpected invocation for "
           + toString());
     }
+
+    @Override
+    public Boolean haveRegisteredExternalStore(String tableName) {
+      throw new UnsupportedOperationException("unexpected invocation for "
+          + toString());
+    }
```

Comment:
I think this just return false not exception. Not sure why other methods throw exceptions. Check with Suranjan.

Return type can better be primitive boolean.